activiti-cloud-audit-service/activiti-cloud-starter-audit/src/test/java/org/activiti/cloud/starter/audit/tests/it/APIEventToEntityConvertersIT.java

```diff
@@ -66,11 +64,14 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.ContextConfiguration;
 import org.springframework.test.context.TestPropertySource;
+import static org.assertj.core.api.Assertions.assertThat;
 
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
 @DirtiesContext
 @TestPropertySource("classpath:application.properties")
+@ContextConfiguration(initializers = ContainersApplicationInitializer.class)
```

Comment:
@robfrank, do you think it would be possible to configure it without adding this annotation to integration tests? It seems a little bit intrusive to me.

Reply:
It's the best way to "pass" the right property to the Apllication Context. I took a look also to an extension of Testcontainers expecially designed for Spring-boot that uses "actuators" and they do the same

Reply:
sorry I approved and forgot this merges automatically, there were some codacy checks failing but we can do another refactoring story